### ISSUE TITLE
Can't play music on Spotify.

### ISSUE THREAD

**Issue by typeling1578** [typeling1578/Thunderfox#1]
When attempting to play Spotify music, the song will skip around and eventually only the first few seconds of music will play.
These problems occur in other Firefox-derivatives or Chromium-derivatives, such as Waterfox, Floorp, ungoogled-chromium, etc., but not in Brave or Vivaldi, for some reason.
If anyone knows how to solve this problem, I would appreciate it if you could let me know.

**Comment by typeling1578** [comment]
Vivaldi Forum: https://forum.vivaldi.net/topic/51466/spotify-web-player-stopped-working

**Comment by typeling1578** [comment]
It appears that the solution to this problem is to sign up with a Widevine certificate provider such as EZDRM, Movidone, Axinom, Uplynk, BuyDRM, etc., get a Widevine certificate and use it to sign binaries.
However, it is very expensive.

**Comment by typeling1578** [comment]
We have found that there are some environments where this problem does not occur.
Further investigation is ongoing.

**Comment by trickypr** [comment]
Forcing gecko to download drm might work to fix this, by setting `media.eme.enabled` to true

**Comment by typeling1578** [comment]
No, even after enabling them, I continue to have problems with Amazon Prime Video, and Spotify.
However, for the Linux version only, Spotify does not have this problem.
